frontend/.eslintrc.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     ecmaFeatures: {
       jsx: true,
     },
-    ecmaVersion: 2018,
+    ecmaVersion: 2020,
```

Comment:
This supports optional chaining.

Reply:
Awesome, I was already wondering why it was always complaining about my new favorite feature :D